src/components/SizeSelector/index.scss

```diff
@@ -0,0 +1,176 @@
+@use '~styles/colors.scss' as colors;
+@use '~styles/breakpoints.scss' as breakpoints;
+@use '~styles/fonts.scss' as fonts;
+
+.container-size-selector {
+  width: 100%;
+  max-width: 15rem;
+
+  display: flex;
+
+  justify-content: center;
+
+  &__size-list {
+    max-width: 100%;
+    overflow-x: hidden;
+
+    display: flex;
+    flex-direction: row;
+    flex-shrink: 0;
+    gap: 2.5rem;
+
+    padding: 5rem 10rem;
+
+    scroll-behavior: smooth;
+    scroll-snap-type: x mandatory;
+  }
+
+  &__card:focus {
+    outline: none;
+  }
+
+  &__card {
```

Comment:
Ainda não temos o cursor pointer.

Coloca um transition aqui pra qunado der hover

Tamo com um problema de visualização aqui, tem um max-width de 12rem ta impedindo o componente de esticar. Lembre-se não podemos ter esse tipo de limitação, no caso ajustar o padding horizontal seria melhor

Reply:
adicionei o cursor pointer;
adicionei o hover em td, n tenho ctz se era isso q vc pediu;
tirei o max-width do card mas precisei colocar um min-width na caixa onde fica os textos pq se um texto for maior q o outro os cards vão ter tamanhos diferentes, n sei se essa é a melhor de resolver isso...